css/card.css

```diff
@@ -399,10 +388,13 @@ section#gacha-controls input {
   transform: translate3d(0.1px, 0.1px, 0.1px);
 }
 #gacha-controls .card-image {
+  position: absolute;
+  top: 0;
+  left: 0;
   box-shadow: 0 10px 20px -5px rgba(0, 0, 0, 0.5);
 }
 #gacha-controls .card-image, #gacha-controls .card-image:before, #gacha-controls .card-image:after {
-  transition: all 0.5s ease;
+  transition: transform 0.5s ease, filter 0.5s ease;
```

Comment:
explicit definition to reduce browser load on possibly transitioning all css properties associated with this element